### PR TITLE
[Bug Fix] Reduce theme upload batch size to prevent timeout

### PR DESCRIPTION
**Pull request by aswamy**
### WHY are these changes introduced?

Part of https://github.com/Shopify/cli/issues/5031

### WHAT is this pull request doing?

A previous [PR](https://github.com/Shopify/cli/pull/4792/files#diff-c4edcc1629cc0813d96396f1661f9f69fcdbece0f346f9d53dc99b6233abd5f1R21) changed the endpoint to upload the themes files from REST API to GraphQL API
- The new endpoint did not cause the issue, but the PR also increased the theme upload batch size
- This is generally fine when using the CLI through regular user login, but fails when requests are being proxied through Theme Access App

### How to test your changes?

- Instead Theme Access App on your store
- Set environment variables in terminal
```
export SHOPIFY_CLI_THEME_TOKEN=<password from app>
export SHOPIFY_FLAG_STORE=<store>
export SHOPIFY_FLAG_FORCE="1"
```
- Run this CLI branch against an existing theme
  - `npx shopify theme push --theme="bug-dec16" --unpublished --verbose --path=<path-to-repo>`
- theme should be uploaded

NOTE: Also try uploading a LOT of asset files AND with large asset files (e.g. download large images in asset folder)

### Measuring impact

How do we know this change was effective? Please choose one:

- [x] n/a - this doesn't need measurement, e.g. a linting rule or a bug-fix
- [ ] Existing analytics will cater for this addition
- [ ] PR includes analytics changes to measure impact

### Checklist

- [x] I've considered possible cross-platform impacts (Mac, Linux, Windows)
- [x] I've considered possible [documentation](https://shopify.dev) changes

### CODE REVIEW

**Review by EvilGenius13 — APPROVED** [review]

**Comment by Waynenabors** [comment]
Just to be clear, does this address the theme pull issue from #5094? I'm currently having issues with push and pull.

**Comment by graygilmore** [comment]
@Waynenabors I wouldn't say with certainty, no. The user that says it's a dupe is not a part of Shopify and may have been guessing.

**Comment by jamesmengo** [comment]
Could we only change these limits when using the theme access app?

File: packages/theme/src/cli/utilities/theme-uploader.ts [hunk 1]
```diff
@@ -18,9 +18,9 @@ type ChecksumWithSize = Checksum & {size: number}
 type FileBatch = ChecksumWithSize[]
 
 // Limits for Bulk Requests
-export const MAX_BATCH_FILE_COUNT = 50
-// 10MB
-export const MAX_BATCH_BYTESIZE = 1024 * 1024 * 10
+export const MAX_BATCH_FILE_COUNT = 20
+// 1MB
+export const MAX_BATCH_BYTESIZE = 1024 * 1024
```
Comment (frandiox) [inline packages/theme/src/cli/utilities/theme-uploader.ts:23]: Should we add a comment here specifying where is the limit coming from? Or perhaps linking this very PR so the context is not lost?

**Review by frandiox — APPROVED** [review]
Thanks for debugging it Alok! Let's ship this for now until we can properly raise the limits 🚀

**Review by jamesmengo — APPROVED** [review]

**Pull request merged**